### PR TITLE
Fixes #271 added express static middleware

### LINKED ISSUE #271: http://localhost:3000 is missing styles, logo, needs express.static middleware
Our current frontend doesn't load properly:

Screen Shot 2019-11-17 at 7 15 53 PM

There are a bunch of issues:

- style/main.css is being sent with a mime type of text/html vs. text/css
- 404 on /dialogue.js
- 404 on img/logo.svg

### PR DESCRIPTION
**Pull request by cindyledev**
Fixes #271 added express static middleware in src/backend/web/routes/index.js

### CODE REVIEW

File: src/backend/web/routes/index.js [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
 
 const router = express.Router();
 
-router.get('/', (req, res) => {
-  res.sendFile(path.join(__dirname, '../../../frontend/index.html'));
-});
+router.use(express.static(path.join(__dirname, '../../../frontend')));
 
-router.use('/admin', admin);
+router.use('/admin', require('./admin'));
```
Comment (miggs125) [inline src/backend/web/routes/index.js]: We are keeping module imports at the top of the file. See [Node style best practices](https://github.com/goldbergyoni/nodebestpractices#-38-require-modules-first-not-inside-functions). Please change it back.

File: src/backend/web/routes/index.js [hunk 2]
```diff
@@ -1,13 +1,10 @@
 const express = require('express');
 const path = require('path');
-const admin = require('./admin');
```
Comment (miggs125) [inline src/backend/web/routes/index.js]: unrelated. Please change this back.

**Comment by cindyledev** [comment]
@miggs125 I changed it back as you requested. Please review again, thank you. :)

**Comment by humphd** [comment]
Awesome!  Testing locally, works great.

**Pull request merged**